### ISSUE TITLE
Error uploading to Twine

### ISSUE THREAD

**Issue by bpstewar** [worldbank/GOSTurban#10]
When running the final commands to upload to PyPi, I get the following errors, and am at a loss.

```
(urban_test) PS C:\WBG\Work\Code\GOST_Urban> python -m twine upload dist/*
Uploading distributions to https://upload.pypi.org/legacy/
Uploading GOST_Urban-0.1.0-py3-none-any.whl
WARNING  Retrying (Retry(total=9, connect=5, read=None, redirect=None, status=None)) after connection broken by
         'SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed
         certificate in certificate chain (_ssl.c:1000)'))': /legacy/
```

**Comment by g4brielvs** [comment]
@bpstewar Thank! If you're using a  World Bank, the error must be because the SSL certificate can't be found. 

However, I'd recommend triggering the publishing to PyPI through a [CI/CD workflow](https://packaging.python.org/en/latest/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/).

**Comment by bpstewar** [comment]
Ok, let's do that for GOST_Urban and gostrocks this week; I need them to complete a reproducibility package.

**Comment by g4brielvs** [comment]
Closed by #12.
